ecmd-core/pyecmd/__init__.py

```diff
@@ -183,6 +183,16 @@ def related_targets(self, target_type, mode=ecmd.ECMD_DYNAMIC_LOOP,
     # alias to match eCMD API spelling
     relatedTargets = related_targets
 
+    def fapi2GetAttr(self, i_id):
```

Comment:
@fenkes-ibm Is there are preferred way to enable/disable this code when we build with and without the fapi2 extension?

Reply:
would the easiest thing be to have #ifdefs and running this through a preprocessor before use?

Reply:
Ah, sorry for not replying right away! I think the easiest thing would be to just leave it as is - if code attempts to use it it will get an AttributeError either way, so I don't think it matters much at which level they get the error, and it will keep #ifdefs out of the code.

As for the API test, I can amend with a small check to see whether the underlying ecmd module exports fapi2GetAttr().